### PR TITLE
code cleanup and re-structure

### PR DESCRIPTION
**Pull request by vivekbhr**
1. code formatting and conda env name formatting.
2. get rid of snakediff and move all Rscript to one folder
3. Fix MultiQC error from HiC pipeline
4. Make documentation build working again [WIP]

### CODE REVIEW

File: bin/snakePipes [hunk 1]
```diff
@@ -17,19 +17,36 @@ def parse_arguments():
     subparsers = parser.add_subparsers(title='Commands',
                                        dest='command')
 
-    infoParser = subparsers.add_parser('info', help='Print the location of the various yaml files')
+    infoParser = subparsers.add_parser('info',
+                                      help='Print the location of the various yaml files')
```
Comment (dpryan79) [inline bin/snakePipes]: Your editor likes to offset things by a space.

Comment (vivekbhr) [reply bin/snakePipes]: what if I say I did it manually :sweat_smile:

Comment (dpryan79) [reply bin/snakePipes]: :man_facepalming:

**Comment by vivekbhr** [comment]
all right.. :+1:

**Pull request merged**